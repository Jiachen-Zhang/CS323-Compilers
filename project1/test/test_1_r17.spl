int test_1_r17(
{
  dec_int = 323
  int a = 0xG;
}
