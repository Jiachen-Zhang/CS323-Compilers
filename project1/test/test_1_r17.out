Error type B at Line 1: Missing closing parenthesis ')'
Error type B at Line 3: Missing semicolon ';'
Error type A at Line 4: unknown lexeme 0xG
Error type B at Line 4: Missing specifier
